autogluon/utils/tabular/ml/models/catboost/catboost_model.py

```diff
@@ -38,6 +39,7 @@ def _set_default_params(self):
             self._set_default_param_value(param, val)
         self._set_default_param_value('random_seed', 0)  # Remove randomness for reproducibility
         self._set_default_param_value('eval_metric', construct_custom_catboost_metric(self.stopping_metric, True, not self.stopping_metric_needs_y_pred, self.problem_type))
+        self._set_default_param_value('allow_writing_files', False)  # Disables creation of catboost logging files during training
```

Comment:
Can you add: "Set this to True in order to keep log files created by catboost during training (these will be found in the directory where AutoGluon stores all of its model files)"

Reply:
Added